src/domain/request/requestExecutor/RequestExecutorFactory.ts

```diff
@@ -0,0 +1,35 @@
+import {LoggerPort} from '@core/domain/common/port';
+import FetchDataUseCase from '@core/domain/common/usecase/FetchDataUseCase';
+import SelectDataUseCase from '@core/domain/common/usecase/SelectDataUseCase';
+import RequestExecutor from '../../../domain/common/port/RequestExecutorPort';
+import InvalidContentTypeError from '../../../domain/common/utils/error/InvalidContentTypeError';
+import UrlRequestExecutor from './UrlRequestExecutor';
+
+class RequestExecutorFactory  {
+    private requestExecutors: RequestExecutor[];
+    constructor(
+        fetchDataUseCase: FetchDataUseCase,
+        selectDataUseCase: SelectDataUseCase,
+        logger: LoggerPort,
+    ) {
+        this.requestExecutors = [new UrlRequestExecutor(fetchDataUseCase, selectDataUseCase, logger)];
```

Comment:
you can move it out of constructor

Reply:
But why? I think havingrequest executors represent state of this class. It doesn't make sense for RequestExecutorFactory to exist if it doesn't have and RequestExecutors.

Are you referring to some JS standard that I'm not aware of?

Reply:
nvm, my bad, sorry